core/assets/bundles/bundle_fr.properties

```
@@ -1274,8 +1274,8 @@ rules.rtsminsquadsize = Taille Minimale d'une Escouade
 rules.rtsmaxsquadsize = Taille Maximale d'une Escouade
 rules.rtsminattackweight = Poids Minimum d'une Attaque
 rules.cleanupdeadteams = Détruire les structures des équipes vaincues (JcJ)
-rules.corecapture = Capture du Noyau lors de sa Destruction
-rules.polygoncoreprotection = Protection Polygonale du Noyau
+rules.corecapture = Capture du Noyau lors de sa destruction
```

```suggestion
rules.corecapture = Capture du Noyau lors de sa Destruction
```
Vaux mieux garder l'accent sur la destruction vu que c important.
https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L1277